Mix lightweight component container (#3)
JohanWork/examples: README.md

1. How to create pipeline components from python functions in jupyter notebook
2. How to compile and run a pipeline from jupyter notebook

#### [Ligtheweight component and container operations](./pipelines/ligtheweight-component-and-container-operations)
Author: [Johan Hansson](https://github.com/JohanWork)

This example covers the following concepts:

> 1. How to create pipeline components from python functions and combining it with a container operations

Reviewer: We we aim for reusable components this could be something in the line of : 
```suggestion
1. How to create pipeline components from python functions and [reusable components](https://www.kubeflow.org/docs/pipelines/sdk/component-development/)
```
